### DISCUSSION TITLE
Specify default file and directory modes for files and directories downloaded from OneDrive

### DISCUSSION THREAD

**Discussion by fishnet37222** [abraunegg/onedrive#1339] [Q&A]
**Is your feature request related to a problem? Please describe.**
Quite often I use OneDrive to store configuration files that I like to share between the various Linux hosts I have.  I find it annoying when I have to manually change a file from mode 600 to mode 644.

**Describe the solution you'd like**
A setting in the config file would satisfy this request.

**Describe alternatives you've considered**
I don't believe there are any alternatives, other than using the proprietary ExpandDrive product which seems to have issues running on systems such as Gentoo and Arch.

**Additional context**
Nothing else to add.

**Comment by abraunegg** [comment]
@fishnet37222 
Please review the usage document - this already exists: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#sync_dir-directory-and-file-permissions
